pytest_fixtures/component/rh_cloud.py

```diff
@@ -15,13 +14,10 @@ def rhcloud_sat_host(satellite_factory):
 
 
 @pytest.fixture(scope='module')
-def rhcloud_manifest_org(rhcloud_sat_host):
+def rhcloud_manifest_org(rhcloud_sat_host, module_entitlement_manifest):
     """A module level fixture to get organization with manifest."""
     org = rhcloud_sat_host.api.Organization().create()
-    manifests_path = rhcloud_sat_host.download_file(file_url=settings.fake_manifest.url['default'])[
-        0
-    ]
-    rhcloud_sat_host.cli.Subscription.upload({'file': manifests_path, 'organization-id': org.id})
+    rhcloud_sat_host.upload_manifest(org.id, module_entitlement_manifest.content)
```

Comment:
why not use SCA/entitlement_manifest_org fixture here instead?

Reply:
In performing the manifester conversions, I have defaulted to not modifying the type of manifest used by tests, under the assumption that component owners should be responsible for making and testing the switch to SCA.

Reply:
@jameerpathan111 ^^